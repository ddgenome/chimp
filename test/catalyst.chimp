## catalytic mechanism, but don't use the kmc lattice
mechanism "catalyst.mech"
## parameter input
parameter "catalyst.par"
## catalyst task
task "catalyst.task"
