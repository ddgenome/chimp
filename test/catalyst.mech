# catalyst without lattice
# adsorption / desorption
A + 2 @ -> k(A_Aads) <- k_arrhenius(A_Ades, E_Ades) @@A;
# surface reaction
@@A -> k_lfer(A_srf, E0_sr, a_srf, DH_srf)
  <- k_lfer(A_srr, E0_sr, 1.0e0 - a_srf, -DH_srf) 2 @B;
@B -> k_arrhenius(A_Bdes, E_Bdes) <- k(A_Bads) B + @;
