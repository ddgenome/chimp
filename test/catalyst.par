# catalyst parameter input file
A_Aads	7.0e2
A_Ades	1.0e11
E_Ades	7.0e4
A_srf	1.0e12
E0_sr	6.1e4
a_srf	5.0e-1
DH_srf	-1.5e4
A_srr	1.0e12
A_Bdes	1.0e11
E_Bdes	6.5e4
A_Bads	6.0e-1
