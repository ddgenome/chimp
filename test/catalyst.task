# -*- text -*-
# test task input
begin model catalyst
  output "catalyst.out"
  begin integrator kmc
    scale 1.0e15
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	@[@@A] = 1.0e-2
	@[@B] = 9.7e-1
      end quantity
      begin output
	(1.0e-2 1.0e-1 1.0e-2)
      end output
      begin reactor batch
	temperature 4.0e2	# K
	pressure 1.0e5		# Pa
	volume 1.0e-5		# m^3
	sites 1.0e19
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
