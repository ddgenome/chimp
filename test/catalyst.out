# catalyst
# x	@	@@A	@B	A	B	steps
0.000000e+00	1.000000e-02	1.000000e-02	9.700000e-01	1.000000e+05	0.000000e+00	0
1.000005e-02	1.700000e-03	1.210000e-02	9.741000e-01	9.144324e+04	1.706768e+04	61872
2.000016e-02	1.500000e-03	1.160000e-02	9.753000e-01	8.335590e+04	3.324125e+04	120451
3.000045e-02	1.700000e-03	1.120000e-02	9.759000e-01	7.563306e+04	4.868804e+04	176393
4.000016e-02	1.700000e-03	1.130000e-02	9.757000e-01	6.831227e+04	6.332963e+04	229416
5.000002e-02	1.400000e-03	1.170000e-02	9.752000e-01	6.149237e+04	7.696776e+04	278805
6.000015e-02	1.800000e-03	1.170000e-02	9.748000e-01	5.495966e+04	9.003540e+04	326125
7.000002e-02	1.600000e-03	1.200000e-02	9.744000e-01	4.887261e+04	1.022084e+05	370208
8.000021e-02	2.000000e-03	1.130000e-02	9.754000e-01	4.327046e+04	1.134149e+05	410795
9.000004e-02	2.200000e-03	1.130000e-02	9.752000e-01	3.807256e+04	1.238118e+05	448445
1.000001e-01	2.000000e-03	1.090000e-02	9.762000e-01	3.324634e+04	1.334631e+05	483403
